cmd/pd-server/main.go

```diff
@@ -136,6 +135,7 @@ func createServerWrapper(args []string) (context.Context, context.CancelFunc, ba
 	if err != nil {
 		log.Fatal("create server failed", errs.ZapError(err))
 	}
+	schedulers.Register()
```

Comment:
Other alternatives might be [Package plugin](https://pkg.go.dev/plugin) or k8s sidecar.